soak_bench_scripts/benchmark.sh

```diff
@@ -1,31 +1,36 @@
 #!/bin/bash
-#set -e
+# set -e
```

Comment:
If this isn't needed, it'd be cleaner to just remove it.

Reply:
Fair point, will update.